apisix/plugins/kafka-logger.lua

```diff
@@ -74,6 +74,15 @@ local schema = {
         inactive_timeout = {type = "integer", minimum = 1, default = 5},
         batch_max_size = {type = "integer", minimum = 1, default = 1000},
         include_req_body = {type = "boolean", default = false},
+        request_body_expr = {
```

Comment:
Would be better to name it as `include_req_body_expr`?

Reply:
done